src/ir/auth_logic/universe_relation_insertion.cc

```diff
@@ -35,6 +35,8 @@ std::string TypeToUniverseName(const datalog::ArgumentType& arg_type) {
       return "isPrincipal";
     case datalog::ArgumentType::Kind::kCustom:
       return absl::StrCat("is", arg_type.name());
+    default:
+      CHECK(false) << "Unexpected variant of Value from IR";
```

Comment:
Fix the error message to match the case here?

Reply:
fixed